### PR TITLE
Gridicons: Update to latest version.

### PR DESCRIPTION
**Pull request by jasmussen**
This version includes a new Chat icon.

See also: https://github.com/Automattic/gridicons/pull/182

To test:

- `make run`
- Verify that gridicons across the board still appear.

Screenshot:

0ea3f368-8c82-11e6-95a5-89be06f17304

The fact that we're updating the entire gridicons react component every time a new icon is added isn't ideal. Perhaps we can get some code-help to [bring home the NPM package](https://github.com/Automattic/gridicons/issues/98)? ;)

### CODE REVIEW

File: client/components/gridicon/docs/example.jsx [hunk 1]
```diff
@@ -19,7 +19,8 @@ module.exports = React.createClass( {
 
 	render: function() {
 		return (
-			<div>
+			<div className="design-assets__group">
```
Comment (mtias) [inline client/components/gridicon/docs/example.jsx]: Ah, we don't need these anymore (we should update the generator cc @retrofox ).

Comment (jasmussen) [inline client/components/gridicon/docs/example.jsx]: I'm adding another icon currently — do I just need to remove the classnames so it's a classless div?

Comment (retrofox) [inline client/components/gridicon/docs/example.jsx]: yes @jasmussen, at least for now. One idea is, in a near future, use a component to wrap the example and implement some utilities.

**Comment by jasmussen** [comment]
Just pushed a new version of the gridicons component. This includes the changes requested to the example file (fixed [upstream as well](https://github.com/Automattic/gridicons/pull/183)), as well as another new icon, offline:

pixels

context

**Pull request merged**